src/lib/mina_ledger/ledger.ml

```diff
@@ -539,10 +539,10 @@ let%test_unit "tokens test" =
     let token_minting =
       mk_forest
         [ mk_node
-            (mk_account_update_body Signature Call token_owner Token_id.default
+            (mk_account_update_body Signature No token_owner Token_id.default
                (-account_creation_fee) )
             [ mk_node
-                (mk_account_update_body None_given Call token_account1
+                (mk_account_update_body None_given No token_account1
```

Comment:
This should be `Parents_own_token`, right? `custom_token_id` is owned by `token_owner`